run_dir/design/base.html

```diff
@@ -43,66 +43,63 @@
         </form>
         <ul class="navbar-nav ml-auto">
           <li class="nav-item dropdown px-1">
-            <a class="nav-link dropdown-toggle" id="nav_production_dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" href="#">Production</a>
+            <a class="nav-link dropdown-toggle" id="nav_production_dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" href="#">Lists</a>
             <div class="dropdown-menu" aria-labelledby="nav_production_dropdown">
-              <h6 class="dropdown-header">General stats</h6>
-              <a class="dropdown-item" href="/applications">Application Counts</a>
-              <a class="dropdown-item" href="/production/cronjobs">Cronjobs List</a>
-              <a class="dropdown-item" href="/flowcells_plot">Flowcells Trends</a>
-              <a class="dropdown-item" href="/data_delivered_plot">Data Delivered Trends</a>
-              <a class="dropdown-item" href="https://ngi-dashboards.scilifelab.se/dbe/">External Dashboard</a>
+              <a class="dropdown-item" href="/deliveries">Deliveries</a>
+              <a class="dropdown-item" href="/flowcells">Flowcells</a>
+              <a class="dropdown-item" href="/projects">Projects</a>
+              <a class="dropdown-item" href="/worksets">Worksets</a>
               <div role="separator" class="dropdown-divider"></div>
-              <h6 class="dropdown-header">Tools</h6>
-              <a class="dropdown-item" href="/sensorpush">Sensorpush</a>
-              <a class="dropdown-item" href="/proj_meta">Compare Project Meta</a>
-              <a class="dropdown-item" href="/instrument_logs">View Instrument Logs</a>
+              <h6 class="dropdown-header">Queues</h6>
+              <a class="dropdown-item" href="/libpooling_queues">Library Pooling</a>
+              <a class="dropdown-item" href="/workset_queues">Pending for Workset</a>
+              <a class="dropdown-item" href="/pools_qpcr">qPCR</a>
+              <a class="dropdown-item" href="/sequencing_queues">Sequencing</a>
+            </div>
+          </li>
+          <li class="nav-item dropdown px-1">
+            <a class="nav-link dropdown-toggle" id="nav_tools_dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" href="#">Tools</a>
+            <div class="dropdown-menu" aria-labelledby="nav_tools_dropdown" role="menu">
               <a class="dropdown-item" href="/barcode">Barcode and Label printing</a>
+              <a class="dropdown-item" href="/proj_meta">Compare Project Meta</a>
               <a class="dropdown-item" href="/pricing_quote">Cost Calculator</a>
+              <a class="dropdown-item" href="/production/cronjobs">Cronjobs List</a>
```

Comment:
Technically a list :smile:

Reply:
I know! So hard to find categories that match the "ranking" of links.